Marlin/UltiLCD2_menu_material.cpp

```diff
@@ -1387,7 +1416,7 @@ void lcd_material_store_material(uint8_t nr)
 
     eeprom_write_byte(EEPROM_MATERIAL_FAN_SPEED_OFFSET(nr), material[active_extruder].fan_speed);
     eeprom_write_float(EEPROM_MATERIAL_DIAMETER_OFFSET(nr), material[active_extruder].diameter);
-    //eeprom_write_block(card.longFilename, EEPROM_MATERIAL_NAME_OFFSET(nr), 8);
+    //eeprom_write_block(card.longFilename, EEPROM_MATERIAL_NAME_OFFSET(nr), 9);
```

Comment:
This line has nothing to do with the added material and should be reverted.

Reply:
Ah, yeah I think I got confused by the `8`, thinking that it would be one for each material. Looking longer at it, it appears that they intended the file name to be 8 words long.